src/components/App.js

```diff
@@ -0,0 +1,15 @@
+// import React, { Component } from "react";
```

Comment:
Why are you still having commented code, is this still useful? If the file isn't useful just remove it.

Reply:
Comments have been removed